Src/UIComponents/Select/Select.css

```diff
@@ -0,0 +1,67 @@
+.container * {
+    box-sizing: border-box;
```

Comment:
@robert-czwartosz @skorpiom please confirm - maybe we could just add this to * instead of having this statement in each file (but let me know if you think anyone uses content-box value)

Reply:
In my opinion we could add this to * in App.css

Reply:
It's great idea.

Reply:
This file was removed from my pull request.